stable/jenkins/README.md

```diff
@@ -78,6 +78,7 @@ The following tables list the configurable parameters of the Jenkins chart and t
 | `rbac.install`                    | Create service account and ClusterRoleBinding for Kubernetes plugin | `false`                                       |
 | `rbac.apiVersion`                 | RBAC API version                     | `v1beta1`                                                                    |
 | `rbac.roleRef`                    | Cluster role name to bind to         | `cluster-admin`                                                              |
```

Comment:
I'd change it to `Role name to bind to`

Reply:
I agree, but only after we add the option to customize Role kind.